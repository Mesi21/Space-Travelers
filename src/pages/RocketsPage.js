import React from 'react';
import AllRockets from '../components/AllRockets';

const RocketsPage = () => (<AllRockets />);

export default RocketsPage;
