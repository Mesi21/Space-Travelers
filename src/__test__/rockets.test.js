import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketsPage from '../pages/RocketsPage';

const rockets = [
  {
    desc: 'Some description',
    key: '1',
    id: '1',
    name: 'Space Traveler 1',
    img: 'https://ahfkjgf.com',
    reserv: false,
  },
];

describe('Test the Rockets Component', () => {
  it('rockets section renders', () => {
    // act
    const { container } = render(
      <Provider store={store}>
        <RocketsPage key={rockets[0].id} rocket={rockets[0]} />
      </Provider>,
    );

    expect(container).toMatchSnapshot();
  });
});