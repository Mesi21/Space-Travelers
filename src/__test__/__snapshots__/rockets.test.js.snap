// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for rendering the RocketsPage Component`] = `
<div>
  <div
    class="container"
  >
    <ul
      class="flex1"
    />
  </div>
</div>
`;

exports[`Test the Rockets Component rockets section renders 1`] = `<div />`;
